### PR TITLE
set of new manual tests for foundries.io

### PR DESCRIPTION
**Pull request by brenwoo**
- tests for all lmp-device-register options
- tests for fleet wide/group/device configuration settings
- test to document fioctl apps vs tags corner case

Signed-off-by: Brendan Wood <brendan.wood@foundries.io>
	               Milosz Wasilewski <milosz.wasilewski@foundries.io>

### CODE REVIEW

**Comment by nareshkamboju** [comment]
@brenwoo 
Please add "Signed-off-by:"  or "Reviewed-by:" before Milosz's name in the commit log.

File: manual/foundries/device-config-overwrite.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    description: "Test for applying device specific config"
+    maintainer:
+        - brendan.wood@foundires.io
+    os:
+        - ubuntu
+        - openembedded
+    scope:
+        - functional
+    devices:
+        -
```
Comment (roxell) [inline manual/foundries/device-config-overwrite.yaml:13]: please keep the device name available or say 'all', do this for all files.

File: manual/foundries/device-config-overwrite.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        - boot up device 
+        - run "lmp-device-register -n<name>"
+        - create a device specific config setting containing "device"
+          run "fioctl devices config set <device> -f<factory> testing-config=device"
+        - check config "cd /run/secrets/" 
+                       
+    expected:
+       - testing-config file is present in /run/secrets/
+       - testing-config contains "device"
+       - device specific setting overwrites fleetwide
```
Comment (roxell) [inline manual/foundries/device-config-overwrite.yaml:30]: different indentations between 'steps:' and 'expected:' please fix, in all files.

File: manual/foundries/device-config-overwrite.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+metadata:
```
Comment (roxell) [inline manual/foundries/device-config-overwrite.yaml:1]: drop the trailing whitespaces, in all files.

**Comment by roxell** [comment]
LGTM. Thank you.

**Comment by nareshkamboju** [comment]
@brenwoo Thanks for your patch.

Author: brenwoo <brendan.wood@foundries.io>
Change this to
Author: Brendan Wood <brendan.wood@foundries.io>

Please edit your .gitconfig settings and commit it again.

**Review by nareshkamboju — APPROVED** [review]
